live-examples/html-examples/text-content/css/figcaption.css

```diff
@@ -4,10 +4,10 @@ figure {
     flex-flow: column;
     margin: 0;
     padding: 5px;
+    width: 22em;
```

Comment:
I would suggest making this `width: 19em;` as `22` causes the output area to scroll vertically.